_config.yml

```diff
@@ -300,6 +300,10 @@ mathjax:
 # Gentie productKey
 #gentie_productKey:
 
+# Gentie hot article
+# gentie_productKey must NOT be null when gentie_hotartical is true!
+#gentie_hotartical: true
```

Comment:
`gentie_hotartical` should be `gentie_hot_articles`. Please don't follow the mistake.